x-pack/plugins/aiops/public/components/log_rate_analysis_results_table/use_view_in_log_pattern_analysis_action.tsx

```diff
@@ -104,7 +104,7 @@ export const useViewInLogPatternAnalysisAction = (dataViewId?: string): TableIte
       return (
         <TableActionButton
           dataTestSubjPostfix="LogPatternAnalysis"
-          iconType="logstashQueue"
+          iconType="logPatternAnalysis"
```

Comment:
This looks good. I think we could update the two icons in the anomalies table too. Could you update the actions for log rate and log pattern analysis as part of this PR too?

<img width="424" alt="Screenshot 2024-11-11 at 09 41 16" src="https://github.com/user-attachments/assets/3a39248a-6471-48c5-a39d-310df9e2d470">

Reply:
Updated both in c26f1ddb76fe09cfe6753196159a6830cb6c5003